src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -361,6 +361,88 @@ def load_arguments(self, _):
+        c.argument('name', id_part=None)
+        c.argument('resource_group', arg_type=resource_group_name_type)
+
+    with self.argument_context('functionapp hybrid-connection add') as c:
+        c.argument('name', arg_type=name_arg_type, id_part=None)
+        c.argument('resource_group', arg_type=resource_group_name_type)
+        c.argument('namespace')
+        c.argument('hybrid-connection')
+
+    with self.argument_context('functionapp hybrid-connection remove') as c:
+        c.argument('name', arg_type=name_arg_type, id_part=None)
+        c.argument('resource_group', arg_type=resource_group_name_type)
+        c.argument('namespace')
+        c.argument('hybrid_connection')
+
+    with self.argument_context('appservice hybrid-connection set-key') as c:
+        c.argument('asp')
+        c.argument('resource_group', arg_type=resource_group_name_type)
+        c.argument('namespace')
+        c.argument('hybrid_connection')
+        c.argument('key-type')
+
+    with self.argument_context('webapp hybrid-connection list') as c:
+        c.argument('name', arg_type=webapp_name_arg_type, id_part=None)
+        c.argument('resource_group', arg_type=resource_group_name_type)
+
+    with self.argument_context('webapp hybrid-connection add') as c:
+        c.argument('name', arg_type=webapp_name_arg_type, id_part=None)
+        c.argument('resource_group', arg_type=resource_group_name_type)
+        c.argument('namespace')
+        c.argument('hybrid-connection')
+
+    with self.argument_context('webapp hybrid-connection remove') as c:
+        c.argument('name', arg_type=webapp_name_arg_type, id_part=None)
+        c.argument('resource_group', arg_type=resource_group_name_type)
+        c.argument('namespace')
+        c.argument('hybrid_connection')
+
+    with self.argument_context('functionapp hybrid-connection list') as c:
+        c.argument('name', id_part=None)
+        c.argument('resource_group', arg_type=resource_group_name_type)
+
+    with self.argument_context('functionapp hybrid-connection add') as c:
+        c.argument('name', arg_type=name_arg_type, id_part=None)
+        c.argument('resource_group', arg_type=resource_group_name_type)
+        c.argument('namespace')
+        c.argument('hybrid-connection')
+
+    with self.argument_context('functionapp hybrid-connection remove') as c:
+        c.argument('name', arg_type=name_arg_type, id_part=None)
+        c.argument('resource_group', arg_type=resource_group_name_type)
+        c.argument('namespace')
+        c.argument('hybrid_connection')
+
+    with self.argument_context('appservice hybrid-connection set-key') as c:
+        c.argument('asp')
+        c.argument('resource_group', arg_type=resource_group_name_type)
+        c.argument('namespace')
+        c.argument('hybrid_connection')
+        c.argument('key-type')
+
+    with self.argument_context('appservice vnet-integration list') as c:
+        c.argument('asp')
```

Comment:
don't use asp - use plan instead, that is what we use in the rest of the commands that use plan as a parameter